$OpenBSD: patch-src_cmd_auxstats_mkfile,v 1.1 2015/12/08 13:05:29 gsoares Exp $

stop linking with kvm(3). OpenBSD's p9p/auxstats no longer uses libkvm.

--- src/cmd/auxstats/mkfile.orig	Mon Dec  7 07:50:40 2015
+++ src/cmd/auxstats/mkfile	Mon Dec  7 07:50:58 2015
@@ -12,6 +12,6 @@ HFILES=\
 
 <$PLAN9/src/mkone
 
-KVM=`{if uname |egrep 'BSD' >/dev/null; then echo -lkvm; fi}
+KVM=`{if uname |egrep 'BSD' | egrep -v 'OpenBSD' >/dev/null; then echo -lkvm; fi}
 LDFLAGS=$LDFLAGS $KVM
 
